jekyll/_cci2/deployment-integrations.md

```diff
@@ -6,20 +6,17 @@ categories: [deploying]
 order: 10
 ---
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/nQo1dngizvA" frameborder="0" allowfullscreen></iframe>
-
-This document describes using the `deploy` step with example instructions in the following sections:
+CircleCI can be configured to deploy to virtually any service. This document provides instructions and examples for the following platforms:
 
 * TOC
 {:toc}
 
-## Deployment Overview 
-
-It is possible to deploy to any service by adding commands to `.circleci/config.yml` and setting secrets on the Project Settings > Environment Variables page of the CircleCI application. Available deployment targets include Azure, Google (App Engine, Container Engine, and Cloud) and many others. 
+## Overview
+{:.no_toc}
 
-Add a job to your `config.yml` to set up conditional deployment for your application. The following example uses a [workflow job filter]({{ site.baseurl }}/2.0/workflows/#using-contexts-and-filtering-in-your-workflows) to check that the current branch is the `master` branch before running any deploy commands. Without this workflow configuration, `<my-deploy-commands>` would be executed every time this job is triggered.
+To deploy your application, add a [job]({{ site.baseurl }}/2.0/jobs-steps/#jobs-overview) to `.circleci/config.yml`. If your deploy job uses any output from previous jobs, you can share that data by [using workspaces]({{ site.baseurl }}/2.0/workflows/#using-workspaces-to-share-data-among-jobs). You will also have to set any necessary secrets on the **Project Settings > Environment Variables** page of the CircleCI application.
```

Comment:
I think it would help to have a similar example that uses workspaces after the first simple example in this doc, so I would move this sentence about workspaces after and then use a modified config example with comments to describe it from the workspaces section.

Reply:
Agree! But this first simple example doesn't actually work in its current form. My next PR is going to add workspaces to this first example, so we have a general sample at the top before getting into any platform-specific instructions.

Reply:
Sounds good